cli/submit.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import json
+import requests
+from getpass import getpass
+
+
+class StudySubmitter:
+    def __init__(self, ENA_AUTH_URL="https://www.ebi.ac.uk/ena/submit/webin/auth/token",
+                 WEBIN_SUBMIT_ENDPOINT ="http://www.ebi.ac.uk/eva/v1/submission/initiate/webin"):
+        self.ENA_AUTH_URL = ENA_AUTH_URL
+        self.WEBIN_SUBMIT_ENDPOINT = WEBIN_SUBMIT_ENDPOINT
+
+    # TODO
+    def submit_with_lsri_auth(self):
+        return NotImplementedError
+
+    # TODO
+    def upload_submission(self, submission_id, submission_upload_url):
+        pass
+
+    def _get_webin_credentials(self):
+        username = input("Enter your ENA Webin username: ")
+        password = getpass("Enter your ENA Webin password: ")
+        return username, password
+
+    def submit_with_webin_auth(self, username, password):
+        print("Proceeding with ENA Webin authentication...")
+
+        headers = {"accept": "*/*", "Content-Type": "application/json"}
+        data = {"authRealms": ["ENA"], "username": username, "password": password}
+        response = requests.post(self.ENA_AUTH_URL, headers=headers, data=json.dumps(data))
+
+        if response.status_code == 200:
+            print("Authentication successful")
+            webin_token = response.text
+            response = requests.post(self.WEBIN_SUBMIT_ENDPOINT + "/" + webin_token)
+            response_json = json.loads(response.text)
```

Comment:
I think this is fine for now but I have seen other makes submission putting the Token in the header (see [BioSamples client](https://github.com/EBIvariation/eva-submission/blob/cb2866a047aca4b589bcca14f3a311436f0eb07f/eva_submission/biosamples_submission.py#L68)

Reply:
Actually this is calling the submission initiation endpoint of eva-submission-ws but the point is still valid. Updated code [here](https://github.com/EBIvariation/eva-sub-cli/pull/6/commits/c1a755ef33c62c002a3de6233e4fb41d02210753)